### PR TITLE
[OSDOCS#9458]: 4.15 RN: Fix wrong acronyms

### PR DESCRIPTION
**Pull request by dulek**
CAPO stands for Cluster API and MAPO for Machine API. This commit fixes incorect explanations of these two acronyms.

This fixes mistakes introduced in #70637.

Version(s):
4.15

Issue:
https://issues.redhat.com/browse/OSDOCS-9302

Link to docs preview:
https://70813--ocpdocs-pr.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-15-release-notes

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by dulek** [comment]
@obrown1205, please review.

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by EmilienM** [comment]
I had a good laugh.

/lgtm

**Review by mandre — COMMENTED** [review]
Actually the whole paragraph doesn't mean anything. The cluster CAPI operator allows provisionning Machines in the same fashion as what OpenShift's Machine API currently allows, but it does not provision a cluster as stated here. @mdbooth could you help produce a more accurate release note for CAPO in cluster CAPI operator?

**Comment by dulek** [comment]
> Actually the whole paragraph doesn't mean anything. The cluster CAPI operator allows provisionning Machines in the same fashion as what OpenShift's Machine API currently allows, but it does not provision a cluster as stated here. @mdbooth could you help produce a more accurate release note for CAPO in cluster CAPI operator?

For the record: release note text comes from here: https://issues.redhat.com/browse/OSASINFRA-3207. It was already tweaked by Matt and @stephenfin.

**Comment by stephenfin** [comment]
Yeah, this text doesn't match what we said. I suspect there's more re-writing to be done here.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 1]
```diff
@@ -61,9 +61,9 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
 ==== CAPO integration into the cluster CAPI Operator (Tech Preview)
 
-If you enable the `TechPreviewNoUpgrade` feature flag, the Coherent Accelerator Processor Interface (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.
+If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.
```
Comment (stephenfin) [inline release_notes/ocp-4-15-release-notes.adoc:64]:
```suggestion
If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
```

File: release_notes/ocp-4-15-release-notes.adoc [hunk 2]
```diff
@@ -61,9 +61,9 @@ This release adds improvements related to the following components and concepts.
...
 
-It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Mail Application Programming Interface (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources and not identical.
+It is now possible to configure the CAPI machine and OpenStack machine resources in a similar fashion to how the Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources and not identical.
```
Comment (stephenfin) [inline release_notes/ocp-4-15-release-notes.adoc:66]:
```suggestion
It is now possible to configure CAPI `Machine` and CAPO `OpenStackMachine` resources in similar fashion to how Machine API (MAPI) resources are configured. It is important to note that the CAPI resources are equivalent to MAPI resources but not identical.
```

File: release_notes/ocp-4-15-release-notes.adoc (hunk 1 above)
Comment (stephenfin) [reply release_notes/ocp-4-15-release-notes.adoc:64]: Note that the `Cluster` and `OpenStackCluster` here are specific API resources provided by CAPI and CAPO respectively. We're *not* talking about "clusters" in the generic sense. As such, I think they need to be formatted as `literals`.

**Review by stephenfin — CHANGES_REQUESTED** [review]

**Comment by stephenfin** [comment]
> Apologies! I received the "lgtm" so I assumed the definitions I had found on IBM's style guide were the correct ones. Thank you for correcting this! How can I help here?

All good :slightly_smiling_face: Please let me know what you think of my suggested replacement text.

**Comment by obrown1205** [comment]
@stephenfin looks good to me! :) /lgtm

**Comment by dulek** [comment]
I think I applied all of the @stephenfin suggestions here.

File: release_notes/ocp-4-15-release-notes.adoc [hunk 3]
```diff
@@ -61,9 +61,9 @@ This release adds improvements related to the following components and concepts.
 [id="ocp-4-15-CAPO-tech-preview-no-upgrade"]
 ==== CAPO integration into the cluster CAPI Operator (Tech Preview)
 
-If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifecycle. The CAPI Operator automatically creates an additional OpenStack cluster and an additional {product-title} cluster for the current {product-title} cluster.
+If you enable the `TechPreviewNoUpgrade` feature flag, the Cluster API (CAPI) Operator deploys the Cluster API Provider for OpenStack (CAPO) and manages its lifestyle. The CAPI Operator automatically creates `Cluster` and `OpenStackCluster` resources for the current {product-title} cluster.
```
Comment (mandre) [inline release_notes/ocp-4-15-release-notes.adoc:64]: is lifestyle correct here? lifecycle seems like a more appropriate word in this context.

Comment (dulek) [reply release_notes/ocp-4-15-release-notes.adoc:64]: Oh yes. Thanks you.

**Review by mdbooth — APPROVED** [review]
/lgtm

**Review by mandre — COMMENTED** [review]
/lgtm

**Comment by obrown1205** [comment]
CC @JoeAldinger

**Review by JoeAldinger — COMMENTED** [review]
This lgtm but I think this PR needs to be against `main` and not `enterprise-4.15`.

**Comment by dulek** [comment]
> This lgtm but I think this PR needs to be against `main` and not `enterprise-4.15`.

Main does not have 4.15 release notes, release notes are only held in release branches: https://github.com/openshift/openshift-docs/blob/main/release_notes/ocp-4-15-release-notes.adoc?plain=1#L7-L8

**Comment by JoeAldinger** [comment]
> > This lgtm but I think this PR needs to be against `main` and not `enterprise-4.15`.
> 
> 
> 
> Main does not have 4.15 release notes, release notes are only held in release branches: https://github.com/openshift/openshift-docs/blob/main/release_notes/ocp-4-15-release-notes.adoc?plain=1#L7-L8

Sorry, you're correct. Would you like this merged?

**Pull request merged**